common/src/chain/config/mod.rs

```diff
@@ -163,6 +187,11 @@ impl ChainConfig {
 const MAX_BLOCK_HEADER_SIZE: usize = 1024;
 const MAX_BLOCK_TXS_SIZE: usize = 524_288;
 const MAX_BLOCK_CONTRACTS_SIZE: usize = 524_288;
+const TOKEN_MIN_ISSUANCE_FEE: Amount = Amount::from_atoms(10_000_000_000_000);
+const TOKEN_MAX_URI_LEN: usize = 1024;
+const TOKEN_MAX_DEC_COUNT: u8 = 18;
+const TOKEN_MAX_TICKER_LEN: usize = 5;
+const TOKEN_MAX_ISSUANCE_ALLOWED: usize = 1;
```

Comment:
I think this constant is misleading. In current implementation number of issuance in tx cannot be changed, thus it shouldn't be part of chain configuration. If we are planning to configure this I suggest adding a TODO explaining what should be fixed.

Reply:
I added the comment